shared/params/config.go

```diff
@@ -202,9 +195,9 @@ var defaultShardConfig = &ShardChainConfig{
 }
 
 var defaultDepositContractConfig = &DepositContractConfig{
```

Comment:
wont this var be invalid since you just removed the type `DepositContractConfig` for this

Reply:
yes, i need to remove it.

Reply:
nvm I still need this. Fixed